app/views/favorites/index.html.erb

```diff
@@ -15,12 +16,20 @@
 <% end %>
 </section>
 
-
-
+<section id="pending-applications">
 <h2>Pending Applications</h2>
+  <% @allpets.pets_with_applications.each do |pet| %>
+    <% if pet.approved == false %>
+    <h4><%= link_to "Name: #{pet.name}", href="/pets/#{pet.id}" %></h4>
+    <% end %>
```

Comment:
This is to ensure that if they are approved, they are not in the pending section.